src/OpenTelemetry/CHANGELOG.md

```diff
@@ -25,6 +25,12 @@
   supported in stable builds.
   ([#5607](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5607))
 
+* `SamplingResult(SamplingDecision decision,
+  IEnumerable<KeyValuePair<string, object>>? attributes)`
+  and `SamplingResult(SamplingDecision decision, string? traceStateString)` constructors
+  accept `null` for `attributes` and `traceStateString` parameters respectively.
+  ([#5614](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5614))
```

Comment:
I think call it a fix. Suggestion...

* Fixed the nullable annotations for the `SamplingResult` constructors to allow `null` being supplied as `attributes` or `traceStateString` which has always been allowed.

Reply:
+1, or even omit the changelog entry?

Reply:
Fixed in d3cf82938a32bf0cb3c484a4390ec65745b48a01